src/main/java/core/basesyntax/dao/UserDao.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.User;
+import java.util.List;
+
+public interface UserDao {
+    void add(User user);
+
+    void delete(User user);
+
+    User getUserById(int id);
+
+    User getUserByName(String name);
+
+    String getUserPassword(String name);
```

Comment:
I don't think we need this method
we could use `getUserByName` and do whatever we want

Reply:
agree